qemu/tests/mmu_basic.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test to disabling radix MMU mode on guest.
+    Steps:
+    1) There are two options, boot up a native(radix)guest or HPT guest.
+    2) Check the MMU mode in the guest.
+    3) Adding disable radix to guest's kernel line directly then reboot guest.
+    4) Check again the MMU mode in the guest.
+    5) Check guest call trace in dmesg log.
+
+
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+    def _check_mmu():
+        mmu_mode = re.search(r'\w{4,}', session.cmd_output(params["mmu_check"]))
+        return mmu_mode.group(0) if mmu_mode else ""
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+
+    if utils_test.check_kernel_cmdline(session, remove_args="disable_radix"):
```

Comment:
Can add `kernel_extra_params_remove = disable_radix` in cfg and not need this step.

Reply:
thank a lot.
correct